src/Login.ts

```diff
@@ -151,6 +179,36 @@ export default class Login {
     }
 }
 
+export interface OidcNativeFlow extends ILoginFlow {
+    type: "oidcNativeFlow";
+    clientId: string;
+}
+/**
+ * Finds static clientId for configured issuer, or attempts dynamic registration with the OP
```

Comment:
This is what `getOidcClientId` does, so it feels like we need to clarify the difference.

It looks like it initialises an `OidcNativeFlow`?